src/software/ai/navigator/navigator.cpp

```diff
@@ -117,33 +99,28 @@ void Navigator::movePointNavigation(const MoveIntent &move_intent,
         }
         case 1:
         {
-            throw std::runtime_error(
-                "Path only contains one point, which is invalid, since it's ambiguous if it's the start or dest or some other point");
-            break;
-        }
-        case 2:
-        {
-            current_destination = (path_points)[1];
+            current_destination = path.calculateValue(1.0);
             auto move           = std::make_unique<MovePrimitive>(
                 move_intent.getRobotId(), current_destination,
                 move_intent.getFinalAngle(),
                 move_intent.getFinalSpeed() *
-                    getCloseToEnemyObstacleFactor((path_points)[1]),
+                    getCloseToEnemyObstacleFactor(path.calculateValue(1.0)),
```

Comment:
perhaps having `path.end()` and `path.start()` methods would help readability a bit here?

Reply:
this is not path end, this is point 1

Reply:
but path start and path end is a good idea

Reply:
is getKnots sufficient? we also know that 1.0 is the end of the spline

Reply:
I think `getKnots` is appropriate here

Reply:
done